modules/tinymce/src/core/main/ts/api/Options.ts

```diff
@@ -311,14 +311,21 @@ const register = (editor: Editor): void => {
   });
 
   registerOption('font_css', {
+    processor: 'string[]'
+  });
+
+  registerOption('allow_extra_mathml_attributes', {
+    processor: 'string[]'
+  });
+
+  registerOption('allow_extra_mathml_elements', {
     processor: (value) => {
-      const valid = Type.isString(value) || Type.isArrayOf(value, Type.isString);
+      const valid = Type.isArrayOf(value, Type.isString);
 
       if (valid) {
-        const newValue = Type.isArray(value) ? value : Arr.map(value.split(','), Strings.trim);
-        return { value: newValue, valid };
+        return { value, valid };
       } else {
-        return { valid: false, message: 'Must be a string or an array of strings.' };
+        return { valid: false, message: 'Must be an array of strings.' };
       }
     },
     default: []
```

Comment:
Why is this not also `string[]`? I don't see why elements should be any different to attributes